roles/blackbox_exporter/tasks/install.yml

```diff
@@ -15,33 +15,56 @@
     createhome: false
   when: blackbox_exporter_user != 'root'
 
-- name: Download blackbox exporter binary to local folder
-  become: false
-  ansible.builtin.unarchive:
-    src: "{{ blackbox_exporter_binary_url }}"
-    dest: "/tmp"
-    remote_src: true
-    creates: "/tmp/blackbox_exporter-{{ blackbox_exporter_version }}.linux-{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}/\
-              blackbox_exporter"
-  register: _download_binary
-  until: _download_binary is succeeded
-  retries: 5
-  delay: 2
-  delegate_to: localhost
-  check_mode: false
-  when: not blackbox_exporter_skip_install
+- name: Discover latest version
```

Comment:
I see it's also missing the ability to use the latest version lookup that the other roles have, perhaps add that while we are at it?

Reply:
Yea, I also noticed this task is named incorrectly in all the roles.